lib/colormaps/xml_to_json.rb

```diff
@@ -22,10 +22,19 @@ def self.xml_to_json(id, url, output_dir)
 
         if entries.size == 0
           # Colormaps 1.2
-          colormaps = xml.xpath("//ColorMaps/ColorMap").find {|map| map.attribute('title').to_s != 'No Data'}
+          no_data = xml.xpath("//ColorMaps/ColorMap").find { |map| map.attribute('title').to_s == 'No Data' }
```

Comment:
`xml.xpath("//ColorMaps/ColorMap")` is repeated 4 times here. Clean it up some so that you aren't making the same call more than once.

I think the `if` can be simplified some too. It looks like you want `colormaps = xml.xpath("//ColorMaps/ColorMap")` either way, so do that outside the `if`, then just check `if !no_data.nil?` or maybe `if no_data.present?` would work